src/awkward/cppyy.py

```diff
@@ -0,0 +1,30 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import awkward as ak
+
+_has_checked_version = False
+
+
+def register_and_check(array=None, cpp_type=None):
```

Comment:
This doesn't actually use `array` or `cpp_type` (nor should it).

```suggestion
def register_and_check():
```